docs/releases/1.9.txt

```diff
@@ -467,6 +471,25 @@ You don't need any of this if you're querying the database through the ORM,
 even if you're using :meth:`raw() <django.db.models.query.QuerySet.raw>`
 queries. The ORM takes care of managing time zone information.
 
+Template tag modules are imported when templates are configured
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`~django.template.backends.django.DjangoTemplates` backend now
+performs discovery on installed template tag modules when instantiated. This
+update enables libraries to be provided explicitly via the ``'libraries'``
```

Comment:
If we are taking about the argument to the backend, then no quotes are needed for 'libraries'. If we are taking about the key in OPTIONS, then we shouldn't call it an argument, I think.